ibl-frontend/frontend-content/src/app/session-list/session/session.component.css

```diff
@@ -32,7 +32,7 @@ h5 {
 
 .fitParTable {
     font-size: 75%;
-    display: table-footer-group;
+    display: table-row;
```

Comment:
Could you add a bit more details here on what this update was addressing? A comment here should suffice. Just trying to understand a bit how this impacted the fixes.

Reply:
This had to deal with the formatting of the tables underneath the graphs of the session page. They can be seen on the bottom portion of this image. 
![image](https://user-images.githubusercontent.com/37351041/141853246-191439ec-2f8b-4ac0-adf6-5bc5274025cf.png)

Reply:
Great, thanks. Makes sense.

Reply:
You're welcome!